Python/Product/VSInterpreters/Interpreter/CondaEnvironmentFactoryProvider.cs

```diff
@@ -228,7 +228,7 @@ class CondaEnvironmentFactoryProvider : IPythonInterpreterFactoryProvider, IDisp
             ForceDiscoverInterpreterFactories();
         }
 
-        private void ForceDiscoverInterpreterFactories() {
+        private async void ForceDiscoverInterpreterFactories() {
```

Comment:
`async void` means that any exception thrown in the body of the method is going to crash VS, right? In particular, `DiscoveryStarted` event handlers and `OnInterpreterFactoriesChanged` can no longer throw - but I don't think that's guaranteed. 

IIRC the usual pattern for this kind of stuff was to make it return `Task`, and then the calling code can use the `DoNotWait()` extension method defined somewhere to explicitly set it to fire-and-forget (and handle exceptions if they arise).

Reply:
Good idea. I'll adjust.